gemrb/core/EnumFlags.h

```diff
@@ -23,6 +23,12 @@
 
 #include <type_traits>
 
+#if __has_attribute(flag_enum)
+#define FLAG_ENUM enum class __attribute__((__flag_enum__))
```

Comment:
@lynxlynxlynx do you know what the gcc version of this is? or another idea to suppress the build warnings when switching on flags?

Reply:
Not there yet nor there seems to be any rush: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=81665

So just add `-Wno-switch` to CXXFLAGS in cmake.